pkg/volume/flexvolume/plugin.go

```diff
@@ -71,21 +123,13 @@ func (plugin *flexVolumePlugin) GetVolumeName(spec *volume.Spec) (string, error)
 	call := plugin.NewDriverCall(getVolumeNameCmd)
 	call.AppendSpec(spec, plugin.host, nil)
 
-	_, err := call.Run()
+	status, err := call.Run()
```

Comment:
I noticed you reverted back to the original implementation here. Was this intentional?